### ISSUE TITLE
how to increase speed GGUF model?

### ISSUE THREAD

**Issue by rakib91221** [city96/ComfyUI-GGUF#8]
how to increase speed GGUF model? 
GGUF model 4 step one image generated time 34 second 6gb model but unet 6gb model generated time 18/19 second

**Comment by city96** [comment]
Could you try on the latest update to see if the speed is better?

**Comment by saviosg** [comment]
I'm getting 8.44s/it on Q4_0, taking 299.63 seconds in average to execute 20 steps, and this number remains stable after multiple generations.
For comparison, I was able to get in average 3s/it on NF4 with Forge previously, with everything else the same.
The machine is a RTX 4050 with 6 GB of VRAM, 8 GB of RAM, and 16 GB of swap.

**Comment by city96** [comment]
@saviosg The model is mapped to system RAM and a copy stays there even after you load it onto the GPU. Even with the smallest quant it'll swap to disk for you because of that.

**Comment by rakib91221** [comment]
> Could you try on the latest update to see if the speed is better?

![11](https://github.com/user-attachments/assets/310cb414-60c1-4a7e-a504-44a05f153e1e)

thank you so much.. now now speed 34/35 second to 18/19 second

**Comment by rakib91221** [comment]
its possible to run without clip i mean text encoder

**Comment by saviosg** [comment]
> @saviosg The model is mapped to system RAM and a copy stays there even after you load it onto the GPU. Even with the smallest quant it'll swap to disk for you because of that.

Indeed. The model doesn't fully fit into RAM, so swap is used. I wonder if K quants would help.
